vendor/engines/bulk_email/app/views/bulk_email/bulk_email/create.html.haml

```diff
@@ -16,20 +16,32 @@
   - if params[:return_path].present?
     = hidden_field_tag :return_path, params[:return_path]
 
-  .span8
-    = label_tag text("bulk_email.recipients")
-    .well= @users.map(&:email).join(", ")
-
-  .span8= f.input :custom_subject, input_html: { class: "input-xxlarge" }
-
-  .span8
-    = f.input :custom_message,
-      as: :text,
-      input_html: { class: "input-xxlarge", rows: 6 }
-
-  .span8
-    = f.submit text("bulk_email.send_mail"),
-      class: "btn btn-primary",
-      data: { disable_with: t("bulk_email.sending") }
-
-    = link_to text("shared.cancel"), bulk_email_cancel_path
+  - if params[:subject_product_id].present?
+    = hidden_field_tag :subject_product_id, params[:subject_product_id]
```

Comment:
Yeah, empties wouldn't hurt and would make the haml a bit nicer if they were gone.